app/apptesting/gamm.go

```diff
@@ -14,22 +14,37 @@ var DefaultAcctFunds sdk.Coins = sdk.NewCoins(
 	sdk.NewCoin("baz", sdk.NewInt(10000000)),
 )
 
-// PrepareUni2PoolWithAssets returns a Univ2 pool with the initial liquidity being the provided balances.
-func (s *KeeperTestHelper) PrepareUni2PoolWithAssets(asset1, asset2 sdk.Coin) uint64 {
+// PrepareUni2PoolWithCoins returns a Univ2 pool with the initial liquidity being the provided balances.
+func (s *KeeperTestHelper) PrepareUni2PoolWithCoins(coin1, coin2 sdk.Coin) uint64 {
```

Comment:
Can we use `PrepareBalancerPoolWithCoins` with 2 coins instead?

Reply:
Didn't know folks would be down for this change as well :) Im totally down to changing this

Reply:
done!